geopyspark/geotrellis/key_conversion.py

```diff
@@ -0,0 +1,157 @@
+'''
+A module to provide facilites for converting between layout keys and spatial objects.  These facilities aim to bridge between the various layouts, ``SpatialKey``s, and geometry.
```

Comment:
In rst, if you have a character that isn't a punctuation mark next to highlighted code it'll mess up the formatting. To get around that, a `\` needs to be added before the character. So ` ``SpatialKey``s ` becomes: ` ``SpatialKey``\s `